base/acme/src/main/java/org/dogtagpki/acme/server/ACMELogoutService.java

```diff
@@ -29,9 +30,10 @@
     @Context
     protected HttpServletRequest servletRequest;
 
+    @GET
```

Comment:
I think some care needs to be taken for `GET /logout` -- one is [prefetching](https://stackoverflow.com/questions/3521290/logout-get-or-post) and the other is -- is a stateless `GET /logout` which immediately logs a user out a DoS vulnerability? Something like `<img src="https://ACME/logout">` would suffice to log the user out if they are already logged in, right? To avoid, you'd need to do a two-step "pre-logout" (get a token and then log out?).

Anyhow both seem avoidable via POST (well, DoS technically occurs anyways but requires user interaction for a cross-site form POST. That's kinda on them at that point).

Reply:
OK, I removed `GET /logout`.

Reply:
Did you deploy this to the testing URL? I see the latter changes there.

For me, login -> logout -> login only prompts me for password once, not twice like I'd expect.

Reply:
(as per IRC, appears to be because of browsers caching basic auth)

Reply:
I added another patch to fix this. Could you try the demo site again? Make sure you force reload the page to load the new JS code.